code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -158,9 +159,10 @@
 	suit_type = "hazmat hardsuit"
 	desc = "An Anomalous Material Interaction hardsuit that protects against the strangest energies the universe can throw at it."
 	icon_state = "science_rig"
-	armor = list(melee = 45, bullet = 5, laser = 45, energy = 80, bomb = 60, bio = 100, rad = 100)
+	armor = list(melee = 45, bullet = 5, laser = 40, energy = 65, bomb = 60, bio = 100, rad = 100)
 	slowdown = 1
 	offline_vision_restriction = 1
+	emp_protection = 40
```

Comment:
I tested EMP resistance in-game, there is a noticable difference between a suit with -20 and 40 EMP resistance. The higher resistance the less likely for the suit to shut for, for a module to be ruined, and for the cell to drain.